crates/runestick/src/stack.rs

```diff
@@ -145,6 +145,15 @@ impl Stack {
         })
     }
 
+    /// Peek the given address without removing it from the stack if it's on the
+    /// top.
+    pub fn address_peek(&mut self, address: InstAddress) -> Result<&Value, StackError> {
```

Comment:
A very minor thing but I try (and often forget) to share prefix of things that are similar instead of suffix. So peek/peek_address, pop/pop_address... I find it helps with code-completion and searching.